specification/security/query_api_keys/QueryApiKeysRequest.ts

```diff
@@ -42,9 +44,20 @@ export interface Request extends RequestBase {
   }
   body: {
     /**
+     * Any aggregations to run over the corpus of returned API keys.
+     * Aggregations and queries work together. Aggregations are computed only on the API keys that match the query.
+     * This supports only a subset of aggregation types, namely: `terms`, `range`, `date_range`, `missing`,
+     * `cardinality`, `value_count`, `composite`, `filter`, and `filters`.
+     * Additionally, aggregations only run over the same subset of fields that query works with.
+     * @aliases aggs */
+    aggregations?: Dictionary<string, AggregationContainer>
```

Comment:
If this only support a subset of the aggregation container we need to build a new container, otherwise its going to confuse users when they try to use those.

Reply:
Hi @Anaethelion ,
I addressed your feedback here, can you please checkup the `APIKeyAggregationContainer` and `APIKeyAggregator`.